spec.html

```diff
@@ -15057,12 +15056,12 @@ <h1>
         <emu-alg>
           1. If IsDetachedBuffer(_O_.[[ViewedArrayBuffer]]) is *true*, return *false*.
           1. If _index_ is not an integral Number, return *false*.
-          1. If _index_ is *-0*<sub>𝔽</sub>, return *false*.
+          1. If _index_ is *-0*<sub>𝔽</sub> or _index_ &lt; *-0*<sub>𝔽</sub>, return *false*.
           1. Let _taRecord_ be MakeTypedArrayWithBufferWitnessRecord(_O_, ~unordered~).
           1. NOTE: Bounds checking is not a synchronizing operation when _O_'s backing buffer is a growable SharedArrayBuffer.
           1. If IsTypedArrayOutOfBounds(_taRecord_) is *true*, return *false*.
           1. Let _length_ be TypedArrayLength(_taRecord_).
-          1. If ℝ(_index_) &lt; 0 or ℝ(_index_) ≥ _length_, return *false*.
+          1. If ℝ(_index_) ≥ _length_, return *false*.
```

Comment:
I am _pretty_ sure this is indeed editorial, but I'd like @syg to confirm. (Specifically, this change means that passing `-1` will now skip the `MakeTypedArrayWithBufferWitnessRecord` step, but per the note about bounds checking not being a synchronizing operation I think that's not observable.)

Reply:
Yeah, I think because it's unordered it is fine.